Feature nav - nawigacja i header finalna wersja

skwirowski opened the pull request:
# Ostatnie fixy, małe poprawki i uzupełnienie informacji.

* Dodane czcionki:
  - Lobster - logo
  - Montserrat - menu & nawigacja
  - Playfair Display - tekst nagłówka
* Dopasowane i wyśrodkowane elementy menu nawigacji w szerokościach mobilnych (z burgerem)
* changed hoover/active color to 
* Zmieniony kolor stanów hover i active w tekście loga i menu nawigacji na kolor zielony #29B200
* Dodane **id** do każdej głównej sekcji dla celów lokalizacyjnych menu nawigacji
  - #feature-info
  - #feature-functions
  - #feature-form
  - #feature-team
* Dodany fragment kodu CSS przed każdą główną sekcją strony - poprawka do offsetu występującego przy użyciu menu nawigacji (spowodowanego position: fixed dla paska menu)
* Drobne poprawki stylizacyjne w pliku CSS
* Dodany div z bootstrapową klasą _jumborton_ w skcji header - cel wyróżnienie nagłówka
* Dodany treść tekstowa headera
----
Jest to wersja ostateczna. Chciałbym, żebyś miał na uwadze to, że dodałem **#id** do każdej głównej sekcji. Nie wiem czy to ma wpływ na kolejność przyjmowania przez Ciebie pull requestów.
----
Pozdrawiam,
Paweł Skwirowski

skwirowski committed:
Adjust menu navigation bar in mobile device sizes, add logo font

skwirowski committed:
Last fixes, small adjustments and completion of missing information

* added fonts
  ^ Lobster - logo
  ^ Montserrat - menu & navigation
  ^ Playfair Display - header text
* adjusted navigation elements positon in mobile widths
* changed hoover/active color to #29B200
* added id for every main section - site navigation purposes
  ^ #feature-info
  ^ #feature-functions
  ^ #feature-form
  ^ #feature-team
* added CSS code snippet before every main section - fix display offset
after using navigation links (caused by fixed navbar)
* small adjustments in CSS file
* added bootstrap jumbotron container in header section
* added header introducing text/message

maciekmp reviewed (changes requested):
Bardzo ładnie to wygląda jednak nie możemy sobie pozwolić na więcej niż 2 czcionki i najlepiej gdyby były one ładowane za pomocą google fonts. Na emulowanym mobile mogę przesunąć stronę horyzontalnie, przed przesunięciem przycisk hamburger jest przy samym ekranie.

skwirowski committed:
Delete local font files, add @import font links, add new logo image

* deleted local all font files
* added @import liknks from google fonts - only 3 fonts are linked
  - Montserrat medium (500) normal,
  - Playfair Display regular (400) italic,
  - Playfair Display bold (700) normal,
* added new logo image so no extra font is needed
* deleted unnecessary CSS code snippets

skwirowski commented:
# Poprawki naniesione. Usunięte lokalne pliki czcionek, dodane linki @import dla google fonts, dodany nowy obrazek z logo
* Wszystkie lokalne pliki z czcionkami zostały usunięte
* Dodane linki @import z czcionkami z google fonts - zlinkowane zostały tylko 3 niezbędne czcionki użyte w projekcie
  - Montserrat medium (500) normal,
  - Playfair Display regular (400) italic,
  - Playfair Display bold (700) normal,
* Dodany nowy obrazek z logo, żeby uniknąć wczytywania kolejnej czcionki
* Usunięte niepotrzebne fragmenty kodu CSS stylujące usunięte elementy HTML'a
* Usunięte komentarze z pliku HTML
---
Już wszystko powinno być dobrze. Projekt został scommitowany i spushowany do gałęzi feature-nav. Tak w sumie to podlinkowane są dwie czcionki, bo dwa ostatnie linki dotyczą tej samej tylko o różnej wadze i rodzaju. Mam nadzieję, że to już jest ostateczna wersja :)

---
Pozdrawiam,
Paweł Skwirowski

skwirowski committed:
Merge branch 'develop' into feature-nav

maciekmp commented:
Jest bardzo fajnie tylko nie mogę zrozumieć jednej rzeczy. Dlaczego jak kliknę na pozycję w menu to zoznacza się ona z szarym tłem i później jak scrolluję po stronie dalej jest szara? Reszta jest ok.
